espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/GetClipboardParams.kt

```diff
@@ -17,5 +17,5 @@
 package io.appium.espressoserver.lib.model
 
 data class GetClipboardParams(
-    val contentType: ClipboardDataType = ClipboardDataType.PLAINTEXT
+    val contentType: ClipboardDataType? = ClipboardDataType.PLAINTEXT
```

Comment:
it can also stay required, although the standard exception thrown is this case is not very information